.gitignore

```diff
@@ -0,0 +1,73 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I suggest adding the source of this files' content. This is because not all of files listed here are relevant for this project and might be confusing if we don't mention this is a standard file.

```suggestion
# Python .gitignore from https://github.com/github/gitignore/blob/HEAD/Python.gitignore
# 
# Byte-compiled / optimized / DLL files
```

Reply:
Done